src/app/project/CreateEditFeature.js

```diff
@@ -121,12 +121,24 @@ define([
         //      sent to toaster
         duplicateActionMsg: 'Can\'t add duplicate actions!',
 
+
         // Properties to be sent into constructor
 
+        // existingData: Object (optional)
+        // {
+        //     category: String,
+        //     retreatment: Boolean,
+        //     geometry: Geometry,
```

Comment:
I suppose my thinking was do as little as possible. Less to go wrong. 

I was thinking of checking if things changed but nuking everything and rewriting would be easier. Probably easier for both of us. Let's make that the plan.